mira/constants.py

```diff
@@ -0,0 +1,34 @@
+"""Constants used across MIRA."""
+
+import pystow
+
+__all__ = ["MODULE", "NODE_HEADER", "EDGE_HEADER"]
+
+#: The PyStow module which allows for system-independent
+#: creation of stable file paths.
+MODULE = pystow.module("mira")
+
+#: The used for the nodes files in the neo4j bulk import
+NODE_HEADER = (
```

Comment:
I think these should live in the `mira.dkg` module. If there is an issue with non-trivial dependencies being imported in the root of `mira.dkg`, then maybe that module can be restructured to move those.